src/Plugin.cc

```diff
@@ -124,9 +124,21 @@ void Plugin::Load(const tinyxml2::XMLElement *_pluginElem)
   this->dataPtr->context->setContextProperty(QString::fromStdString(filename),
       this);
 
+  // TODO perhaps find a more generic way of locating these resource files
```

Comment:
Yeah we need  to do that, using build-defined paths (i.e. not `cwd`) and making sure it is cross-platform.